frontend/packages/core/src/card.tsx

```diff
@@ -77,7 +78,7 @@ interface CardHeaderProps {
 const CardHeader = ({ avatar, children, title, summary = [] }: CardHeaderProps) => {
   return (
     <StyledCardHeaderContainer>
-      <StyledCardHeader container wrap="nowrap" alignItems="center" spacing={2}>
```

Comment:
opted to set custom padding instead for the grid items b/c by default `spacing` adds padding all over  and we only need a bit of right/left padding so that the items don't "touch" each other when the screen size gets smaller

i.e. to avoid this
<img width="157" alt="Screen Shot 2021-09-01 at 9 42 45 PM" src="https://user-images.githubusercontent.com/39421794/131767887-0ef87d48-dccb-4eab-8591-2a3774ba722a.png">

with a bit of padding
<img width="165" alt="Screen Shot 2021-09-01 at 9 44 11 PM" src="https://user-images.githubusercontent.com/39421794/131767969-99d5efb2-9e78-484d-a1d2-aa1161390689.png">